test/app/features/escape_manual/data/model/escape_manual_fixtures.dart

```diff
@@ -39,15 +71,76 @@ const escapeManualEntityFixture = EscapeManualEntity(
       title: 'group',
       tasks: [
         EscapeManualTaskEntity(
-          id: 'id',
-          type: 'type',
+          id: '1',
+          type: 'checkbox',
+          description: 'description',
+          isEditable: false,
+          isDone: false,
+          userInputValue: 'userInputValue',
+        ),
+        EscapeManualTaskEntity(
+          id: '3',
+          type: 'checkbox',
           description: 'description',
+          userInputValue: null,
           isDone: false,
+          isEditable: false,
+        ),
+      ],
+    ),
+    EscapeManualTasksSectionEntity(
+      title: 'group 2',
+      tasks: [
+        EscapeManualTaskEntity(
+          id: '2',
+          type: 'checkbox_contact',
+          description: 'description',
           isEditable: true,
+          isDone: true,
+          userInputValue: null,
+        ),
+      ],
+    ),
+  ],
+);
+
+final updatedEscapeManualEntityFixture = EscapeManualEntity(
+  assistant: EscapeManualAssistantEntity(
+    explanation: 'explanation',
+    action: EscapeManualAssistantActionEntity(
+      text: 'text',
+      quizSession: QuizSessionModel(
+        sessionId: 'sessionId',
+      ),
+    ),
+  ),
+  sections: [
+    EscapeManualTasksSectionEntity(
+      title: 'group',
+      tasks: [
+        EscapeManualTaskEntity(
+          id: '1',
+          type: 'checkbox',
+          description: 'description',
+          isEditable: false,
+          isDone: true,
           userInputValue: 'userInputValue',
```

Comment:
Faz sentido esse valor num `checkbox` não editável?

Reply:
Esses valores são aleatórios para serem utilizados pelos testes unitários. Sinceramente eu não sei se faria sentido, pois o `isEditable` foi mantido apenas para ficar semelhante ao retorno da API e não é utilizado para mais nada no aplicativo, talvez o melhor seria remover essa propriedade?

Reply:
Referência a `userInputValue`

Reply:
Coloquei um valor aleatório somente para testar a lógica, pois ainda não tem a funcionalidade que atribui um valor real/válido para essa propriedade, de qualquer maneira mudei para `null` e quando a funcionalidade for integrada deixo o tipo que faça mais sentido.